list_portfolio_fills.go

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2023-present Coinbase Global, Inc.
```

Comment:
update this to 2024-present since this was added this year

Reply:
fixed